python/common/python/math/__init__.java

```diff
@@ -0,0 +1,840 @@
+package python.math;
```

Comment:
This file and package needs to be re-organised per #517, since there are no sub-modules.